PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -724,21 +758,26 @@ struct HfCandidateCreatorXic0Omegac0 {
       KFParticle kfPosPr(kfTrack0, kProton);
       KFParticle kfNegPi(kfTrack1, kPiMinus);
       KFParticle kfNegKa(kfTrackBach, kKMinus);
+      KFParticle kfNegPirej(kfTrackBach, kPiMinus); // rej
       KFParticle kfPosPi(kfTrack0, kPiPlus);
       KFParticle kfNegPr(kfTrack1, kProton);
       KFParticle kfPosKa(kfTrackBach, kKPlus);
+      KFParticle kfPosPirej(kfTrackBach, kPiPlus); // rej
 
       KFParticle kfBachKaon;
       KFParticle kfPos;
       KFParticle kfNeg;
+      KFParticle kfBachrej; // rej
```

Comment:
If I get it properly, this "rej" always refers to a pion, i.e. you consider the bachelor particle of the Omega cascade as a pion, right? If this is the case, I'd suggest calling it `kfBachPionRej` or similar.

Also: I would try to make it clear in the code with some comments that your intention is to reject cases in which the candidate has a an inv. mass compatibler to Xi (bachelor pion) instead of Omega (bachelor kaon)